exporter/api/query_handlers.go

```diff
@@ -4,21 +4,20 @@ import (
 	"encoding/hex"
 	"fmt"
 
-	spectypes "github.com/bloxapp/ssv-spec/types"
-
 	specqbft "github.com/bloxapp/ssv-spec/qbft"
+	spectypes "github.com/bloxapp/ssv-spec/types"
 	"go.uber.org/zap"
```

Comment:
please sort the import

Reply:
@nivBlox 

I usually sort imports according this rule:
1. std packages
2. packages from other public repositories
3. packages from the current repository

Do you want to sort them according to another rule?

Reply:
yeah exactly